src/index.html

```diff
@@ -10,5 +10,31 @@
   <body>
     <div id="main"></div>
     <script src="./js/index.js"></script>
+    <ul>
```

Comment:
Javascript should be included at the end of the document.
The reason is JS should be executed after the HTML DOM is loaded.

You can put all the content within `<div id="main"></div>`